hazelcast/src/test/java/com/hazelcast/jet/core/MemberReconnectionStressTest.java

```diff
@@ -116,5 +120,14 @@ public void test() {
             }
             sleepMillis(100);
         }
+        //Test finished, close the threads
+        try {
+            terminated.set(true);
```

Comment:
I think we can make `terminated` local variable and remove the `after` method completely.

Reply:
Ok